.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
```

Comment:
It might be worth adding `sudo: false` to [speed things up a little](https://docs.travis-ci.com/user/trusty-ci-environment/#Container-based-with-sudo%3A-false).